testnet/stacks-node/src/tests/neon_integrations.rs

```diff
@@ -742,10 +748,10 @@ fn liquid_ustx_integration() {
 
     let publish = make_contract_publish(&spender_sk, 0, 1000, "caller", caller_src);
 
-    let replaced_txid = submit_tx(&http_origin, &publish);
+    let replaced_txid = submit_tx(&http_origin, &publish, TransactionAnchorMode::Any);
```

Comment:
Why does `Any` match the previous behavior? I would have thought it was maybe `OnChainOnly`? I thought the new behavior you were adding was more permissive, but this makes it seem like the old behavior was more permissive.

Reply:
`Any` does not match the previous behavior. However, based on the discussion in the linked issue, we want `Any` to be the new default behavior.

Reply:
discussed offline: as long as we are sure some "decision maker" understands that we are shipping this product change

Reply:
OK. as discussed offline as sure as we are aware the "product decision making body" understands this change is happening.